### PR TITLE
Repo set up and addition of kueue directory

### PR DESCRIPTION
**Pull request by Fiona-Waters**
This PR adds updates to the repo readme, a template runbook and an OWNERS file for the repo.
In a separate commit I have added the kueue directory which contains runbooks for kueue alerts and an OWNERS file.

### CODE REVIEW

File: alerts/kueue/kueue-pod-down.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+```bash
+oc -n redhat-ods-applications get pods -l app.kubernetes.io/name=kueue
+```
+
+2. If the pod is not running, look at the pod's logs/events to see what may be causing the issues. Please make sure to grab the logs/events so they can be shared with the engineering team later:
+
+```bash
+# Check pod logs 
+oc -n redhat-ods-applications logs -l app.kubernetes.io/name=kueue --prefix=true
```
Comment (sutaakar) [inline alerts/kueue/kueue-pod-down.md]: This command shows just last 10 lines.
Do we rather want to retrieve whole log file? For that user can use:
```
oc -n redhat-ods-applications logs -l app.kubernetes.io/name=kueue --prefix=true --tail=-1
```

File: alerts/kueue/pending-workload-pods.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+2. Get further details on the pod.
+```bash
+pod=< pod-name >
+oc describe pod $pod -n $namespace
+```
+
+3. Review the pod logs and determine why it is in a pending state. 
+```bash
+oc logs $pod -n $namespace
```
Comment (sutaakar) [inline alerts/kueue/pending-workload-pods.md:30]: Pending Pods don't have a log as they didn't start. Though trying to retrieve a Pod log may hint some issues:
```
ksuta@ksuta-thinkpadp1gen4i:~$ oc logs $pod -n $namespace
Error from server (BadRequest): container "httpd" in pod "example" is waiting to start: trying and failing to pull image
```

Comment (Fiona-Waters) [reply alerts/kueue/pending-workload-pods.md:30]: I think it is still valuable to run this command - is this phrasing better?
```
3. Attempt to retrieve the pod logs if any are available and determine why it is in a pending state. 
```

Comment (sutaakar) [reply alerts/kueue/pending-workload-pods.md:30]: Sounds better

**Review by sutaakar — APPROVED** [review]

**Review by Ygnas — APPROVED** [review]

**Pull request merged**